src/main.js

```diff
@@ -54,6 +55,7 @@ Vue.component('frappe-control', FrappeControl);
 Vue.component('f-button', Button);
 Vue.component('indicator', Indicator);
 
+Vue.use(FullCalendar);
```

Comment:
import FullCalendar where it is used, not globally. In this case, this should go in Calendar.vue

Reply:
Cool, will clean the code and make the necessary changes and update soon.